mnemosyne/core/beam.py

```
@@ -989,6 +989,59 @@ def init_beam(db_path: Path = None):
+                LEFT JOIN memory_embeddings me ON me.memory_id = wm.id
+                WHERE me.memory_id IS NULL
+                  AND wm.content IS NOT NULL
+                  AND length(wm.content) > 0
+            """).fetchall()
+            for row in _backfill_missing:
+                try:
+                    vec = _embeddings.embed([row["content"]])
+                    if vec is not None and len(vec) == 1:
+                        _store_working_embedding(conn, row["id"], vec[0])
+                except Exception:
+                    pass  # best-effort; log noise would repeat per-row
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)